src/pages/components/Tab/Tab.mdx

```diff
@@ -33,23 +33,21 @@ const MyComponent = () => {
 
 <Playground title="Default">
   <Playground.Preview>
-    <Tab onchange={(idx) => alert(`Choosen Tab: ${idx}`)}>
-      <Tab.Item title="Profile" active>
-        Tab content 1
-      </Tab.Item>
-      <Tab.Item title="Notifications">Tab content 2</Tab.Item>
-      <Tab.Item title="Settings">Tab content 3</Tab.Item>
-    </Tab>
+    <Preview />
   </Playground.Preview>
   <Playground.Code>
     ```jsx
-    <Tab onchange={(idx) => alert(`Choosen Tab: ${idx}`)}>
-      <Tab.Item title="Profile" active>
-        Tab content 1
-      </Tab.Item>
-      <Tab.Item title="Notifications">Tab content 2</Tab.Item>
-      <Tab.Item title="Settings">Tab content 3</Tab.Item>
-    </Tab>
+    <Tab onchange={(idx) => (this.idx = idx)}>
```

Comment:
@acneidert can you just format the indentation?

Reply:
Done! XD